### DISCUSSION TITLE
Beo6 API

### DISCUSSION THREAD

**Discussion by NilanPerera** [larsbaunwall/Beolyd5#5] [Q&A]
Hi,
Do you know how the Beo6 handles two way communicates with the BeoMaster 5? I think its via XMPP but I cant find any details. I have a Beo6 and want to mimic the BeoMaster 5 but I cant find any information on how it communicates. Any help would be really appreciated

**Comment by djerik** [comment]
Have you tried using Wireshark to capture the network traffic?

I am looking for a Beo6 remote, but still a bit too expensive for just playing with it. If someone stumbles on a cheap one, even with broken screen or similar, let me know, so I can do some analysis on it
